CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Changed: [#5559](https://github.com/ethereum/aleth/pull/5559) Update peer validation error messages.
 - Fixed: [#5562](https://github.com/ethereum/aleth/pull/5562) Don't send header request messages to peers that haven't sent us Status yet.
 - Changed: [#5568](https://github.com/ethereum/aleth/pull/5568) Improve rlpx handshake log messages and create new rlpx log channel.
+- Changed: [#5574](https://github.com/ethereum/aleth/pull/5574) Simplify rlpx handshake logging calls and log abridged genesis hashes in peer validation output.
```

Comment:
Group `Changed` items together.
Although I probably wouldn't add an item for this at all, logging change is very internal and abridged hashes are minor

Reply:
Agreed, changes have trivial amount of user impact. Will remove the entry for these changes.